.github/workflows/checks.yml

```diff
@@ -14,8 +14,6 @@ jobs:
         run: npm install
       - name: Build
         run: npm run build
-      - name: Test
-        run: npm run test:ci
```

Comment:
These changes are irrelevant to https://github.com/firebase/firebase-js-sdk/issues/6747